README.md

```diff
@@ -12,10 +12,116 @@
 
 ## Installation
 
+To install the SignalFlow Python client library, open a terminal and run the
+following command:
+
 ```console
 pip install signalflow-client-python
 ```
 
+## Run a SignalFlow computation
+
+The following example allows you run a SignalFlow computation from the command
+line.
+
+1. Create a file called ``signalflow_demo.py`` that includes the following
```

Comment:
Kind of a nit, but since frameworks often have naming conventions, the statement that the reader create a file called `x` might suggest to some readers that the specific name or some part of it has a special meaning and is required for this example. Another option here would be to suggest that there is no naming convention, e.g.

> Create a file, e.g. `app.py`, containing the following